test/test-h5.jl

```diff
@@ -0,0 +1,86 @@
+testlisteq = (a, b) -> @test Set(a) == Set(b)
+
+include(joinpath(@__DIR__, "../workflow/scripts/h5.jl"))
```

Comment:
maybe better to use "ice-floe-tracker-pipeline/workflow/scripts/h5.jl"? Or will the relative path always work, even in containers?

Reply:
I think this should work as well as long as the files are not moved around.